SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -198,8 +220,10 @@ void ASpatialDebugger::CreateWorkerRegions()
 	for (const FWorkerRegionInfo& WorkerRegionData : WorkerRegions)
 	{
 		AWorkerRegion* WorkerRegion = GetWorld()->SpawnActor<AWorkerRegion>(SpawnParams);
-		WorkerRegion->Init(WorkerRegionMaterial, WorkerRegionData.Color, WorkerRegionOpacity, WorkerRegionData.Extents, WorkerRegionHeight,
-						   WorkerRegionVerticalScale);
+		FString WorkerInfo = FString::Printf(TEXT("You are looking at virtual worker number %d\n%s"), WorkerRegionData.VirtualWorkerID,
+											 *WorkerRegionData.WorkerName);
+		WorkerRegion->Init(WorkerRegionMaterial, WorkerCombinedMaterial, WorkerInfoFont, WorkerRegionData.Color, WorkerRegionOpacity,
+						   WorkerRegionData.Extents, WorkerRegionHeight, WorkerRegionVerticalScale, WorkerInfo, bInEditor);
```

Comment:
Instead of passing `bInEditor` around, couldn't we just deduce it?

Reply:
I haven't found a way to deduce if the class is created in the editor or at runtime. However, as the UCanvasRenderTarget2D->OnCanvasRenderTargetUpdate() callback is only triggered at runtime this bool can be removed.